rdflib/extras/infixowl.py

```diff
@@ -383,17 +387,71 @@ def __init__(self, identifier=None, graph=None):
             except Exception:  # pragma: no cover
                 pass  # pragma: no cover
 
+    def snc(self, graph, bnc=False):
+        """
+        Take terms referencing this individual as a subject and
+        add them to the provided graph.
+        """
+        for fact in self.factoryGraph.triples((self.identifier, None, None)):
+            graph.add(fact)
+
+        return graph
+
+    def bnc(self, graph, bnc=False):
```

Comment:
Similar to `snc`, it seems like the boolean `bnc` is not used in this function, so I would like to remove it if possible.

```suggestion
    def bnc(self, graph):
```